en/full-node.md

```diff
@@ -0,0 +1,362 @@
+
+* Table of contents
+{:toc}
+
+<ul class="reportissue"><li><a href="https://github.com/bitcoin/bitcoin.org/issues/new">Report An Issue</a></li></ul>
+<ul class="editsource"><li><a href="https://github.com/bitcoin/bitcoin.org/tree/master/en/full-node.md">Edit On GitHub</a></li></ul>
+
+</div></div>
+<div markdown="1" class="toccontent">
+
+## What Is A Full Node?
+
+A full node is a program that fully validates transactions and blocks.
+Almost all full nodes also help the network by accepting transactions
+and blocks from other full nodes, validating those transactions and
+blocks, and then relaying them to further full nodes.
+
+Most full nodes also serve lightweight clients by allowing them to
+transmit their transactions to the network and by notifying them when a
+transaction affects their wallet. If not enough nodes perform this
+function, clients won't be able to connect through the peer-to-peer
+network---they'll have to use centralized services instead.
+
+Many people and organizations volunteer to run full nodes using spare
+computing and bandwidth resources---but more volunteers are needed to
+allow Bitcoin to continue to grow.  This document describes how you can
+help and what helping will cost you.
+
+## Costs And Warnings
+
+Running a Bitcoin full node comes with certain costs and can expose you
+to certain risks. This section will explain those costs and risks so you
+can decide whether you're able to help the network.
+
+### Special Cases
+
+Miners, businesses, and privacy-conscious users rely on particular
+behavior from the full nodes they use, so they will often run their own
+full nodes and take special safety precautions. This document does not
+cover those precautions---it only describes running a full node to help
+support the Bitcoin network in general.
+
+Please [consult an expert](/en/community) if you need help setting up
+your full node correctly to handle high-value and privacy-sensitive
+tasks.
+
+### Secure Your Wallet
+
+It's possible and safe to run a full node to support the network and use
+its wallet to store your bitcoins, but you must take the same
+precautions you would when using any Bitcoin wallet.  Please see the
+[securing your wallet page](/en/secure-your-wallet) for more
+information.
+
+### Minimum Requirements
+
+Bitcoin Core full nodes have certain requirements. If you try running a
+node on weak hardware, it may work---but you'll likely spend more time
+dealing with issues. If you can meet the following requirements, you'll
+have an easy-to-use node.
+
+* Desktop or laptop hardware running recent versions of Windows, Mac OS
+  X, or Linux.
+
+* 50 gigabytes of free disk space
+
+* 2 gigabytes of memory (RAM)
+
+* Broadband Internet connection
+
+* 6 hours a day that your full node can be left running. (You can do
+  other things with your computer while running a full node)
+
+### Possible Problems
+
+* **Legal:** Bitcoin use is [prohibited or restricted in some
+  areas](http://bitlegal.io/)
```

Comment:
It's a single-paragraph bullet point, so I don't end it with a period.  See the other bullet points in this list and other lists in the doc.  I open to suggestions to change the style, although I think we had a similar discussion before about sentences in tables where we decided to omit the trailing period.  (I don't care either way, as long as we agree.)